### DISCUSSION TITLE
Magick.NET-Q8-AnyCPU - Version 7.14.5 failed in Windows Store Certification

### DISCUSSION THREAD

**Discussion by edsonmata** [dlemstra/Magick.NET#879] [Help]
Has any version of Magick solved this microsoft store certification problem ?

![image](https://user-images.githubusercontent.com/5148183/111355987-abad9c80-8666-11eb-8afd-391673bd7cab.png)

My Version:
![image](https://user-images.githubusercontent.com/5148183/111356352-0ba44300-8667-11eb-95cc-05b443bc5909.png)

**Comment by dlemstra** [comment]
Magick.NET will fail the Windows Store Certification because it uses certain native calls that are not supported there.
